macros/SpecName.ejs

```diff
@@ -848,24 +836,24 @@ var specList = {
         url  : 'http://dev.w3.org/2006/webapi/selectors-api2/'
     },
     'Shadow DOM':{
-        name : 'Shadow DOM',
-        url  : 'https://www.w3.org/TR/shadow-dom/'
+        name : 'DOM (Shadow trees)',
```

Comment:
I think we won't be able to remove it entirely as some stuff is still not upstreamed into DOM/HTML. As such I will go through and update those pages to point to upstreamed stuff where appropriate, and then revert this edit so that the remaining stuff (like elementFromPoint) still points to the Shadow DOM spec for now.